tools/gn

```diff
@@ -333,6 +337,10 @@ def parse_args(args):
   parser.add_argument('--bitcode', default=False, action='store_true',
                       help='Enable bitcode for iOS targets.')
 
+  parser.add_argument('--stripped', default=True, action='store_true',
```

Comment:
Can you change the flag name to something that indicates that this only affects the contents of `flutter.jar` on Android?  (e.g. `--flutter-jar-unstripped`)

Reply:
I think the fact that it's not on iOS is probably more of a bug than a feature right? I think we should ideally not create one off features and stripping symbols is a generic feature.